intTests/test0063_detect_vacuity/test.saw

```diff
@@ -0,0 +1,13 @@
+// This should fail, as the preconditions are contradictory
+
+let bad_spec = do {
+    a <- crucible_fresh_var "a" (llvm_int 32);
+    crucible_precond {{ a == 0 }};
+    crucible_precond {{ a == 1 }};
+    crucible_execute_func [crucible_term a];
+    crucible_postcond {{ a == 2 }};
+};
+
+m <- llvm_load_module "test.bc";
+
+crucible_llvm_verify m "double_imp" [] true bad_spec z3;
```

Comment:
does `fails (crucible_llvm_verify ...)` work? Then `if` in bash would be unnecessary.

Reply:
Ah, I should actually edit the comment at the top of the file, because the command does not fail upon finding a contradiction, it just emits a warning in the output but keeps on performing the verification.